Documentation/CRDs/Object-Storage/ceph-object-store-crd.md

```diff
@@ -216,13 +214,21 @@ security:
       VAULT_SECRET_ENGINE: kv
       VAULT_BACKEND: v2
     # name of the k8s secret containing the kms authentication token
-    tokenSecretName: rgw-vault-token
+    tokenSecretName: rgw-vault-kms-token
+  s3:
+    connectionDetails:
+      KMS_PROVIDER: vault
+      VAULT_ADDR: http://vault.default.svc.cluster.local:8200
+      VAULT_BACKEND_PATH: rgw
+      VAULT_SECRET_ENGINE: transit
+    # name of the k8s secret containing the kms authentication token
+    tokenSecretName: rgw-vault-s3-token
 ```
 
 For RGW, please note the following:
 
-* `VAULT_SECRET_ENGINE` option is specifically for RGW to mention about the secret engine which can be used, currently supports two: [kv](https://www.vaultproject.io/docs/secrets/kv) and [transit](https://www.vaultproject.io/docs/secrets/transit). And for kv engine only version 2 is supported.
-* The Storage administrator needs to create a secret in the Vault server so that S3 clients use that key for encryption
+* `VAULT_SECRET_ENGINE` option is specifically for RGW to mention about the secret engine which can be used, currently supports two: [kv](https://www.vaultproject.io/docs/secrets/kv) and [transit](https://www.vaultproject.io/docs/secrets/transit). And for kv engine only version 2 is supported with AWS-SSE:KMS, only transit engine supported for AWS-SSE:S3.
+* The Storage administrator needs to create a secret in the Vault server so that S3 clients use that key for encryption for AWS-SSE:KMS
```

Comment:
```suggestion
* The Storage administrator needs to create a secret in the Vault server so that S3 clients use that key for encryption with `AWS-SSE:KMS`.
```